.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "\U0001F4DC Documentation"
+description: Suggest an idea for this project
```

Comment:
I don't think that this is clear.

Reply:
`Suggest your documentation concern or idea`